recipes/rust/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import PureWindowsPath
+import os.path
+import sys
+
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.files import copy, get, rename, replace_in_file
+from conan.tools.microsoft import is_msvc
+
+
+class RustConan(ConanFile):
+    name = "rust"
```

Comment:
I wonder if `rust` or `rustc` makes the most sense. Not sure if others have opinions? Make sure the `topics` attribute below doesn't include the exact name of the package.

Reply:
I actually started with rustc. Since it includes both the compiler and the standard library I settled on rust.

Reply:
I think `rust` is better, because looking into Rust's GH, is fits to https://github.com/rust-lang/rust/archive/refs/tags/1.64.0.tar.gz and there is no `rustc` project listed.

Reply:
Ex rust dev (I only last 6 months)

Rust is the language or runtime and rustc is the toolchain https://doc.rust-lang.org/rustc/what-is-rustc.html as ref plus https://github.com/rust-lang/rust folders 

If this is only the compiler then I would call this it `rustc` ... personally I would reserve `rust` for a package that includes this plus cargo since you could bootstrap a dev environment but that's a bit overkill